doc/doc-requirements.txt

```diff
@@ -1,16 +1,16 @@
 joblib
+memory_profiler
+mne
+nilearn
 numpy
 numpydoc
 matplotlib
 pandas
 pillow
+pyvista
```

Comment:
These dependencies are extravagant to me. We should have them optional, as this is used only in examples.